includes/acl/core/memory_utils.h

```diff
@@ -167,19 +167,34 @@ namespace acl
 		template<typename Type>
 		struct safe_underlying_type<Type, false> { using type = Type; };
 
+		template<typename DstType, typename SrcType, bool isreal = false>
+		struct is_float_impl {
```

Comment:
structs should have their opening brace on the following line to match the surrounding code